packages/go_router/lib/src/route.dart

```diff
@@ -437,6 +438,9 @@ class GoRoute extends RouteBase {
   /// ```
   final ExitCallback? onExit;
 
+  /// Whether the path is case sensitive or not.
+  final bool caseSensitive;
```

Comment:
I was wondering if it was something we intend to keep supporting. If not, maybe I add a `@Deprecated` ?

Reply:
let's keep it for now. I think a certain portion of the websites are still case insensitive. we can evaluate again in the future.